<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editor_add_band</class>
 <widget class="QDialog" name="editor_add_band">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>30</y>
     <width>610</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Inserir Banda</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lb_firstname">
            <property name="text">
             <string>Nome:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ln_firstname"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>Data inicio:</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QDateEdit" name="dt_birthday"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Musicos:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Search:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="ln_search"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableView" name="tb_search_musicien"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="pb_add_musicien">
                <property name="text">
                 <string>NÃ£o Listado?</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="pb_search">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Procurar</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="pb_add">
                <property name="text">
                 <string>Adicionar</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_remove">
                <property name="text">
                 <string>Remover</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tb_selected_musicians"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Biografia:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="ln_biography"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>150</height>
               </size>
              </property>
              <property name="text">
               <string>Fotografia:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_upload_foto">
              <property name="text">
               <string>Upload</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="idk_2">
              <property name="text">
               <string>IDK:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ln_idk">
              <property name="text">
               <string>Apelido</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="pb_close">
              <property name="text">
               <string>Fechar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_add_band">
              <property name="text">
               <string>Adicionar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_close</sender>
   <signal>clicked()</signal>
   <receiver>editor_add_band</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>840</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>783</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
